packages/core/strapi/src/core-api/service/collection-type.ts

```diff
@@ -1,6 +1,6 @@
 import { propOr } from 'lodash/fp';
 import { contentTypes } from '@strapi/utils';
-import type { CoreApi, Schema } from '@strapi/types';
+import type { CoreApi, Schema, Entity } from '@strapi/types';
```

Comment:
I think it makes sense to leave it as 'Entity' rather than 'Document'in v4, it might be confusing if not.

Reply:
Agree, let's add a note maybe?

Reply:
done!